tasmota/settings.h

```diff
@@ -853,8 +859,8 @@ typedef union {                            // Restricted by MISRA-C Rule 18.4 bu
     uint16_t http_init : 1;
     uint16_t shutter_moved : 1;
     uint16_t shutter_moving : 1;
-    uint16_t spare11 : 1;
-    uint16_t spare12 : 1;
+    uint16_t eth_connected : 1;
+    uint16_t eth_disconnected : 1;
     uint16_t spare13 : 1;
```

Comment:
There is an issue with shutters here. I don't like the shutter solution and I will see if this can be solved first to merge your PR.

In short, ifdef USE_SHUTTERS changes the amount of bits tested based on the assumptions the last two bits are for shutter status. This changes with your implemntation.

Reply:
The bits are not persistent so I suggest the following:
- Insert your entries just above the ``http_init`` bit.
- Increment both ``MAX_RULES_FLAG`` by two.
- Update ``RulesEvery50ms()`` by insert your rule text above the ``HTTP`` entries. So they become 8 and 9. HTTP becomes 10 and the optional shutter ones become 11 and 12.